### PR TITLE
Allow cw-orch wasm compilation without features

### PR DESCRIPTION
**Pull request by Buckram123**
This PR aims to allow adding derive attributes for wasm targets, by not generating any code.

Interface can be removed as well. 
Users can also use the `#[cfg(not(target_arch = "wasm32"))]` to shield the interface instead of the interface feature

### CODE REVIEW

File: contracts/mock_contract/src/custom_resp.rs [hunk 1]
```diff
@@ -73,7 +72,3 @@ pub fn migrate(_deps: DepsMut, _env: Env, msg: MigrateMsg) -> StdResult<Response
         ))
     }
 }
-
-#[cfg(feature = "interface")]
-#[cw_orch::interface(InstantiateMsg, ExecuteMsg, QueryMsg, MigrateMsg)]
-struct MockContract;
```
Comment (Buckram123) [inline contracts/mock_contract/src/custom_resp.rs:79]: @Kayanski we can keep it, without any feature/target attributes

Comment (Kayanski) [reply contracts/mock_contract/src/custom_resp.rs]: Fixed

File: cw-orch/src/lib.rs [hunk 2]
```diff
@@ -6,37 +6,49 @@
...
-#[cfg(feature = "daemon")]
-pub extern crate tokio;
+#[cfg(not(target_arch = "wasm32"))]
 #[cfg(feature = "daemon")]
 pub mod daemon;
 
+#[cfg(not(target_arch = "wasm32"))]
 mod error;
 #[cfg(feature = "osmosis-test-tube")]
 pub mod osmosis_test_tube;
```
Comment (CyberHoward) [inline cw-orch/src/lib.rs:20]: We should either arch-flag all the exports or only the non-featured ones (if we assume a user doesn't enable the feature in their wasm env). 

Right now daemon is arch-flagged but test-tube is not.

Comment (Buckram123) [reply cw-orch/src/lib.rs:20]: I think arch-flagging features would work as well!

Comment (Kayanski) [reply cw-orch/src/lib.rs:20]: Oups, that's a mistake yes

Comment (Kayanski) [reply cw-orch/src/lib.rs:20]: flaggin features doesn't seem to work : https://github.com/rust-lang/cargo/issues/1197

**Pull request merged**